nwb_conversion_tools/datainterfaces/behavior/deeplabcut/deeplabcutdatainterface.py

```diff
@@ -0,0 +1,54 @@
+"""Authors: Saksham Sharda, Cody Baker, Ben Dichter."""
+
+from ....basedatainterface import BaseDataInterface
+from ....utils import FilePathType
+
+try:
+    from dlc2nwb.utils import write_subject_to_nwb, auxiliaryfunctions
+
+    HAVE_DLC2NWB = True
+except ImportError:
+    HAVE_DLC2NWB = False
+
+
+class DeepLabCutInterface(BaseDataInterface):
+    """Data interface for DeepLabCut datasets"""
+
+    def __init__(self, dlc_file_path: FilePathType, config_file_path: FilePathType):
+        """
+        Interface for writing DLC's h5 files to nwb using dlc2nwb.
+
+        Parameters
+        ----------
+        dlc_file_path: FilePathType
+            path to the h5 file output by dlc.
+        config_file_path: FilePathType
+            path to .yml config file
+        """
+        if "DLC" not in dlc_file_path or not dlc_file_path.endswith(".h5"):
+            raise IOError("The file passed in is not a DeepLabCut h5 data file.")
+        assert HAVE_DLC2NWB, "to use this datainterface: 'pip install dlc2nwb'"
+        self._config_file = auxiliaryfunctions.read_config(config_file_path)
+        super().__init__(dlc_file_path=dlc_file_path, config_file_path=config_file_path)
+
+    def get_metadata(self):
+        metadata = dict(
+            NWBFile=dict(session_description=self._config_file["Task"], experimenter=[self._config_file["scorer"]]),
+            Subject=dict(subject_id="ind1"),
```

Comment:
where are you gettinb "ind1" from?

Reply:
If we use this, every subject for a dandiset will have the same name, "ind1"

Reply:
currently, its expected of the user to change the metadata(add `session_start_time`, and modify `subject_id`).
But yes, I agree that if we have the `subject_name` as an optional argument to the `init`, it prompts the user to care to specify the subject name. 
As for the default `subject_id`, I would prefer it to not be a constant ("ind1") but construct it from the video name: `subject_id=f"subject_{video_name}"`. Since every video file will be unique to a subject, if the user does not specify the subject name for a bunch of nwb files each will have a unique subject name.